sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubConsumer/EventHubConsumerLiveTests.cs

```diff
@@ -1862,33 +1862,6 @@ await using (var client = new EventHubClient(connectionString))
             }
         }
 
-        /// <summary>
-        ///   Verifies that the <see cref="EventHubConsumer" /> is able to
-        ///   connect to the Event Hubs service and perform operations.
-        /// </summary>
-        ///
-        [Test]
-        [Ignore("Expected behavior currently under discussion")]
-        public async Task ConsumerCanReceiveWhenClientIsClosed()
-        {
-            await using (EventHubScope scope = await EventHubScope.CreateAsync(1))
-            {
-                var connectionString = TestEnvironment.BuildConnectionStringForEventHub(scope.EventHubName);
-
-                await using (var client = new EventHubClient(connectionString))
-                {
-                    var partition = (await client.GetPartitionIdsAsync()).First();
-
-                    await using (EventHubConsumer consumer = client.CreateConsumer(EventHubConsumer.DefaultConsumerGroupName, partition, EventPosition.Latest))
-                    {
-                        client.Close();
-
-                        Assert.That(async () => await consumer.ReceiveAsync(1, TimeSpan.Zero), Throws.Nothing);
-                    }
-                }
-            }
-        }
-
```

Comment:
Per discussion, the service team requests that we have a 1:1 relationship between the client and AMQP connections.   The client owns the connection, so when it is closed, any child consumer/producer will no longer be able to operate, by design.